lib/grpc_rest.rb

```diff
@@ -203,7 +203,18 @@ def send_gruf_request(klass, service_obj, method, request)
         active_call: nil,
         message: request
       )
-      handler.send(method.to_sym)
+      Gruf::Interceptors::Context.new(gruf_interceptors(request)).intercept! do
```

Comment:
Just wondering for my own knowledge, do you know if spans have been created by datadog at this point in execution?

Reply:
There should be one, yes.